go.mod

```diff
@@ -47,9 +47,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20220506065057-8b7839c62700+incompatible
-	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
```

Comment:
Why are there changes to go.mod and go.sum in a docs PR?

Reply:
There were some lint issues involving the go files but I ended up resolving them a different way. I'll remove these.